apps/src/codebridge/FilePreview/HTMLPreview.tsx

```diff
@@ -84,13 +128,17 @@ export const HTMLPreview = () => {
     // When we switch levels, clear the source so the preview does not show outdated content.
     setDebouncedSource(undefined);
     setIsLevelLoading(true);
-    setCurrentFile(DEFAULT_START_HTML_FILE);
```

Comment:
I removed this here because it was not getting executed on page load - only when levels switched. So I moved to `useEffect` below on line 191 triggered on level load/level id change.